lib/resources/AccessTokens.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var TelnyxResource = require('../TelnyxResource');
+var telnyxMethod = TelnyxResource.method;
+
+module.exports = require('../TelnyxResource').extend({
+  path: 'access_tokens',
+  basePath: '/v2-beta',
```

Comment:
Is the plan to name-space the API at the path level for beta endpoints moving forward?

Reply:
that's something we are testing out with video. We thought it would be better so that we are clear, from the path, that this is a beta endpoint that will be deprecated.